config/rhoai/kustomization.yaml

```diff
@@ -0,0 +1,28 @@
+# Adds namespace to all resources.
+namespace: opendatahub
```

Comment:
if this is for rhoai, i guess the namespace should be "redhat-odh-applications"

Reply:
Does it matter? Isn't it overwritten by the ODH operator?

The RHDS fork repository is synced as a mirror, so we can set it to `redhat-odh-applications` if that's what's expected.

Reply:
since it was changed i thought you wanna match whats for rhoai, but yes, operator transform namespace so not a big deal